lib/commands/network.js

```diff
@@ -38,7 +38,8 @@ commands.setNetworkConnection = async function (type) {
   });
   if (!airplaneMode) {
     await this.wrapBootstrapDisconnect(async () => {
-      await this.adb.setWifiAndData({wifi, data});
+      await this.adb.setWifiState(wifi);
+      await this.adb.setDataState(data);
```

Comment:
Yes. Sorry. I didn't see that it was `setWifiState` and `setDataState`, not the `setWifiAndDataState` or whatever.